### PR TITLE
Update what-are-phone-system-auto-attendants.md

### PR DESCRIPTION
**Pull request by CLYVR**
Added information on partial name searches for voice enabled directory searches.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/what-are-phone-system-auto-attendants.md [hunk 1]
```diff
@@ -132,6 +132,8 @@ Callers can say names in the following formats:
 |LastName + FirstName |Full  |Marble Amos |Amos Marble |
 |FirstName |Full |Amos |Press or say 1 for Amos Marble  <br/> Press or say 2 for Amos Jones |
 |LastName |Full |Marble |Press or say 1 for Amos Marble  <br/> Press or say 2 for Ben Marble |
+|FirstName or LastName |Partial |Mar |Press 1 or say for Mary Marble  <br/> Press or say 2 for Mary Jones  <br/> Press or say 3 for Amos Marcus |
```
Comment (JohanFreelancer9) [inline Teams/what-are-phone-system-auto-attendants.md:135]:
```suggestion
|FirstName or LastName |Partial |Mar |Press or say 1 for Mary Marble  <br/> Press or say 2 for Mary Jones  <br/> Press or say 3 for Amos Marcus |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@MikePlumleyMSFT PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by MikePlumleyMSFT** [comment]
@CLYVR this article is being retired. Replaced by [Plan for Teams auto attendants and call queues](https://docs.microsoft.com/en-us/microsoftteams/plan-auto-attendant-call-queue) and [Auto attendant and call queue dialing and voice recognition reference](https://docs.microsoft.com/en-us/microsoftteams/dial-voice-reference).

**Comment by CLYVR** [comment]
Closing - updated in new articles.

**Pull request closed without merging**